src/ubx.cpp

```diff
@@ -1901,6 +1904,10 @@ GPSDriverUBX::payloadRxAddMonVer(const uint8_t b)
 
 			if (fwver_str != nullptr) {
 				GPS_INFO("u-blox firmware version: %s", fwver_str + strlen("FWVER="));
+				const char *fwver_val = strrchr(fwver_str, ' ')
```

Comment:
```suggestion
				const char *fwver_val = strrchr(fwver_str, ' ');
```
this doesn't compile